README.markdown

```diff
@@ -26,14 +26,14 @@ Documentation
 The bulk of the documentation is stored in the `Resources/doc/index.md`
 file in this bundle:
 
-[Read the Documentation for master](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Resources/doc/index.md)
+[Read the Documentation for this version](Resources/doc/index.md)
 
-[Read the Documentation for 1.2.0 (for Symfony 2.0.x)](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/1.2.0/Resources/doc/index.md)
+[Read the old Documentation for 1.2.0 (for Symfony 2.0.x)](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/1.2.0/Resources/doc/index.md)
```

Comment:
@weaverryan what is the relation for that ? FOSUserBundle 1.3 is maintained. This link is about FOSUserBundle 1.2.0, which was the version of the bundle targetting Symfony 2.0.x